src/core.h

```diff
@@ -20,6 +20,14 @@
 
 #include "argon2.h"
 
+#if defined(_MSC_VER)
+#define ALIGN(n) __declspec(align(16))
+#elif defined(__GNUC__) || defined(__clang)
+#define ALIGN(x) __attribute__((__aligned__(x)))
+#else
+#define ALIGN(x)
+#endif
+
```

Comment:
This macro has been removed in commit 7c57b3f08c8630c521916cad329c022cb38fca63. You should not reintroduce it, unless you need to use it.